src/ctelnet.cpp

```diff
@@ -2606,9 +2694,25 @@ bool cTelnet::loadReplay(const QString& name, QString* pErrMsg)
         }
         loadingReplay = true;
         if (mudlet::self()->replayStart()) {
-            // TODO: consider moving to a QTimeLine based system...?
-            // This initiates the replay chunk reading/processing cycle:
-            loadReplayChunk();
+            std::pair results = preparseReplayFile();
+            if (Q_LIKELY(std::get<0>(results))) {
+                mReplayHasFaultyFormat = std::get<1>(results);
```

Comment:
```suggestion
                mReplayHasFaultyFormat = modifiedFormat;
```

... or maybe even `auto [ok, mReplayHasFaultyFormat]` will work? try.

Reply:
I'll stick with the initial suggestion - it'll probably only be there for a short while anyhow - I am currently working on a further development to read the actual replay data from the `QMap` that this PR currently puts it into - in which case the flag will never need to escape the `preparseReplayFile()` method.